NtupleTools/test/make_ntuples_cfg.py

```diff
@@ -261,13 +262,13 @@
 
 # Need the global tag for geometry etc.
 envvar = 'mcgt' if options.isMC else 'datagt'
-#GT = {'mcgt': '74X_mcRun2_asymptotic_v4', 'datagt': '74X_dataRun2_reMiniAOD_v1'}
-#
-#process.GlobalTag.globaltag = cms.string(GT[envvar])
+GT = {'mcgt': '76X_mcRun2_asymptotic_RunIIFall15DR76_v1', 'datagt': '76X_dataRun2_16Dec2015_v0'}
+
+process.GlobalTag.globaltag = cms.string(GT[envvar])
 
 from Configuration.AlCa.GlobalTag import GlobalTag
-GT = {'mcgt': 'auto:run2_mc', 'datagt': 'auto:run2_data'}
-process.GlobalTag = GlobalTag(process.GlobalTag, GT[envvar], '')
+#GT = {'mcgt': 'auto:run2_mc', 'datagt': 'auto:run2_data'}
+#process.GlobalTag = GlobalTag(process.GlobalTag, GT[envvar], '')
 
```

Comment:
You should leave default as auto (unless you have a good reason) and pass the specific one you want using the globaltag parameter.